packages/frontend/src/components/DashboardTabs/index.tsx

```diff
@@ -419,7 +437,7 @@ const DashboardTabs: FC<DashboardTabsProps> = ({
                         tab={activeTab}
                         dashboardTiles={dashboardTiles}
                         onClose={() => setDeletingTab(false)}
-                        opened={isDeletingTab}
+                        opened={isDeletingTab && dashboardTabs.length > 1}
```

Comment:
`dashboardTabs` seems not optional, but I can see this in many places `dashboardTabs?.[0];` , should we check if  dashboardTabs is optional here (and other places) ? like `dashboardTabs?.length`